react-state/my-app/src/ToggleButton.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+
+export default function ToggleButton({ text, color }) {
+  const [isClicked, setIsClicked] = useState(false);
+  console.log('useState', isClicked);
+  const handleClick = () => {
+    console.log('before setter', isClicked);
+    setIsClicked(!isClicked);
+    console.log('after setter', isClicked);
+  };
+  if (isClicked) {
+    return (
+      <button
+        onClick={handleClick}
+        style={{ backgroundColor: color, color: 'black' }}>
+        {text}
+      </button>
+    );
+  }
+  return (
+    <button
+      onClick={handleClick}
+      style={{ backgroundColor: 'white', color: 'black' }}>
+      {text}
+    </button>
+  );
```

Comment:
The only thing different in these two return statements is the backgroundColor. So there is a lot of code duplication. In React, you want to write JSX only once, so you want to structure your code to avoid duplication. See if you can do that for this component.